doc/BorderRouter.rst

```diff
@@ -103,29 +107,30 @@ Packets currently identified for slow-path are:
 
 - SCMP traceroute packets
 
-Configuration
----------------
-
-The configuration of the border router will remain in the border router toml file.
-The following configuration entries are added:
-
 Pool size
 ^^^^^^^^^^^
 
-The size of the packet buffer pool of the receivers can be configured.
-An optimal value could be derived by calculating the maximum number of packets in-flight through the system:
+The pool size will be set by calculating the maximum number of packets in-flight through the system:
 
 .. code-block:: text
 
     pool_size := numReaders * readBatch + numProcessors * (processorQueueSize + 1) + numWriters * (writerQueueSize + writeBatchSize)
 
+Configuration
+---------------
+
+The configuration of the border router will remain in the border router toml file.
+The following configuration entries are added:
+
 Number of processing routines (N)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 By configuring the number of processing routines one can specify the number of goroutines that are able
 to process packets in parallel.
 An optimal value should be derivable from the maximum latency of processing and forwarding a packet,
 the speed of the network interfaces and the number of available CPU cores.
+Since having more processing routines than number of CPU cores would just lead to swapping, the maximum
+number of processing routines are limited by the environment variable GOMAXPROCS.
```

Comment:
Doesn't GOMAXPROCS just limit the number of threads that a go application can use?
Wouldn't this mean that we also need to account for the read and write go routines?
Otherwise we still have context switches between go routines.

Reply:
Ok, it would be good to add those thoughts to the design document then.

Reply:
If I think about it I still somehow don't like the idea of using GOMAXPROCS at all. I think it would still be better to just make it configurable in the config file.